plugfest-3-2023/signed_badge_example-p3.json

```
@@ -0,0 +1,43 @@
+{
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1",
+    "https://purl.imsglobal.org/spec/ob/v3p0/context.json"
```

From what I learned from @kayaelle the context URL now needs to be:
```suggestion
    "https://purl.imsglobal.org/spec/ob/v3p0/context-3.0.2.json"
```

PR #47 updates the other example credentials.

---

Updating this context (URL) alters the credential data model (N-Quads) as follows; this will need to be re-signed:
```diff
--- signed_badge_example-p3-06d380a.nq	2023-09-30 10:21:16.527252812 -0400
+++ signed_badge_example-p3-3.0.1.nq	2023-09-30 10:20:59.215462439 -0400
@@ -2 +2 @@
-<did:key:z6MkjnTsAhSmGMxbGrHqGqoubDpdWyPbPTNWpcYBNfWXMXgD> <https://purl.imsglobal.org/spec/vc/ob/vocab.html#Achievement> <urn:uuid:ac254bd5-8fad-4bb1-9d29-efd938536926> .
+<did:key:z6MkjnTsAhSmGMxbGrHqGqoubDpdWyPbPTNWpcYBNfWXMXgD> <https://purl.imsglobal.org/spec/vc/ob/vocab.html#achievement-0> <urn:uuid:ac254bd5-8fad-4bb1-9d29-efd938536926> .
@@ -4 +4 @@
-<did:key:z6MknTHVAhsRyNzBG6h5wjTrg76UGUaBoXS5Vs9Apfr1fbCj> <https://schema.org/name> "Jobs for the Future (JFF) Labs"^^<https://www.w3.org/2001/XMLSchema#string> .
+<did:key:z6MknTHVAhsRyNzBG6h5wjTrg76UGUaBoXS5Vs9Apfr1fbCj> <https://schema.org/name> "Jobs for the Future (JFF) Labs" .
@@ -8 +8 @@
-<urn:uuid:7b2605ae-e89d-4689-9d9b-fe5395e286df> <https://schema.org/name> "JFF x vc-edu PlugFest 3 Interoperability"^^<https://www.w3.org/2001/XMLSchema#string> .
+<urn:uuid:7b2605ae-e89d-4689-9d9b-fe5395e286df> <https://schema.org/name> "JFF x vc-edu PlugFest 3 Interoperability" .
@@ -15,3 +15,3 @@
-<urn:uuid:ac254bd5-8fad-4bb1-9d29-efd938536926> <https://purl.imsglobal.org/spec/vc/ob/vocab.html#image> <https://w3c-ccg.github.io/vc-ed/plugfest-3-2023/images/JFF-VC-EDU-PLUGFEST3-badge-image.png> .
-<urn:uuid:ac254bd5-8fad-4bb1-9d29-efd938536926> <https://schema.org/description> "This wallet supports the use of W3C Verifiable Credentials and has demonstrated interoperability during the presentation request workflow during JFF x VC-EDU PlugFest 3."^^<https://www.w3.org/2001/XMLSchema#string> .
-<urn:uuid:ac254bd5-8fad-4bb1-9d29-efd938536926> <https://schema.org/name> "JFF x vc-edu PlugFest 3 Interoperability"^^<https://www.w3.org/2001/XMLSchema#string> .
+<urn:uuid:ac254bd5-8fad-4bb1-9d29-efd938536926> <https://purl.imsglobal.org/spec/vc/ob/vocab.html#Image> <https://w3c-ccg.github.io/vc-ed/plugfest-3-2023/images/JFF-VC-EDU-PLUGFEST3-badge-image.png> .
+<urn:uuid:ac254bd5-8fad-4bb1-9d29-efd938536926> <https://schema.org/description> "This wallet supports the use of W3C Verifiable Credentials and has demonstrated interoperability during the presentation request workflow during JFF x VC-EDU PlugFest 3." .
+<urn:uuid:ac254bd5-8fad-4bb1-9d29-efd938536926> <https://schema.org/name> "JFF x vc-edu PlugFest 3 Interoperability" .
@@ -19 +19 @@
-_:c14n0 <https://purl.imsglobal.org/spec/vc/ob/vocab.html#narrative> "Wallet solutions providers earned this badge by demonstrating interoperability during the presentation request workflow. This includes successfully receiving a presentation request, allowing the holder to select at least two types of verifiable credentials to create a verifiable presentation, returning the presentation to the requestor, and passing verification of the presentation and the included credentials."^^<https://www.w3.org/2001/XMLSchema#string> .
+_:c14n0 <https://purl.imsglobal.org/spec/vc/ob/vocab.html#narrative> "Wallet solutions providers earned this badge by demonstrating interoperability during the presentation request workflow. This includes successfully receiving a presentation request, allowing the holder to select at least two types of verifiable credentials to create a verifiable presentation, returning the presentation to the requestor, and passing verification of the presentation and the included credentials." .
```

---

Applied in bb2c6f04a513c31454b844a1bfeaab6769162c85